examples/documentation/documentation.bal

```diff
@@ -37,3 +37,11 @@ public type DummyRecord record {
     string fieldOne;
     int fieldTwo;
 };
+
+// This is the documentation attachment of the `dummyFunction` function.
+# This function returns the fieldTwo of type `DummyRecord`.
```

Comment:
Shall we add an empty line between this and the param/return?

```ballerina
// This is the documentation attachment of the `dummyFunction` function.
# This function returns the fieldTwo of type `DummyRecord`.
#
# + recordValue - Parameter of type `DummyRecord`.
# + return - Return the fieldTwo of type `DummyRecord`.
```

Reply:
Done.